runhouse/main.py

```diff
@@ -846,6 +907,23 @@ def stop(
         subprocess.run(RAY_KILL_CMD, shell=True)
 
 
+@server_app.command("status")
+def server_status(
+    cluster_name: Optional[str] = typer.Argument(
+        None, help="A *saved* remote cluster object to restart."
+    ),
+):
+    """Check the HTTP server status on the cluster."""
+    logger.debug("Checking the server status.")
+
+    current_cluster = get_cluster_or_local(cluster_name=cluster_name)
+    try:
+        current_cluster.client.check_server()
+        console.print("Server is up.")
```

Comment:
thoughts on something like this? 
```suggestion
        console.print("f"Server is up and running on host: {current_cluster.client.host} and port {current_cluster.client.port}.")
```